### PR TITLE
Add a GraphQL schema

### PR DESCRIPTION
**Pull request by lornajane**
Also update the library version

### CODE REVIEW

File: redocly.yaml [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-        - lang: 'Python'
-        - lang: 'JavaScript'
-          label: 'JS'
-        - lang: 'Go'
-        - lang: 'Java'
-        - lang: 'C#'
+openapi:
+  showExtensions: true
+  jsonSamplesExpandLevel: 5
+  codeSamples:
```
Comment (adamaltman) [inline redocly.yaml:4]: I wouldn't configure `codeSamples`. The zero config is pretty good. We used to not have any defaults.

File: package.json [hunk 2]
```diff
@@ -23,6 +23,6 @@
   },
   "homepage": "https://github.com/Redocly/apidocs-starter#readme",
   "dependencies": {
-    "@redocly/redoc": "^0.93.0"
+    "@redocly/redoc": "^0.93.3"
```
Comment (adamaltman) [inline package.json]:
```suggestion
    "@redocly/redoc": "^0.94.2"
```
The `theme` removal was in 0.94.0.

File: redocly.yaml (hunk 1 above)
Comment (lornajane) [reply redocly.yaml:4]: I just wanted _something_ to show that the config file does things, and actually there's not that much for openapi until you get into the weeds with it. Happy to swap out the codesamples settings, but it seemed like an easy placeholder for people to find and edit themselves as they learn.

Comment (adamaltman) [reply redocly.yaml:4]: Maybe remove the languages from it?

Comment (lornajane) [reply redocly.yaml:4]: Our defaults are too good, people won't edit the config! The whole point is that by including the settings, they will immediately want to change them and that's what I'm trying to unlock here (why do you think the theme is an unbearable yellow? It's an invitation to change)

Comment (lornajane) [reply redocly.yaml:4]: (resolving the conversation so I can publish the content, but totally happy to have more changes, I don't think these are specifically called out in the related started guide text)

**Review by JLekawa — APPROVED** [review]

**Pull request merged**